cmd/kubeadm/app/constants/constants.go

```diff
@@ -238,7 +238,7 @@ var (
 	SupportedEtcdVersion = map[uint8]string{
 		8:  "3.0.17",
 		9:  "3.1.10",
-		10: "3.1.10",
+		10: "3.2.10",
```

Comment:
this _might_ be bumped to 3.2.11 or something higher as well before things are merged in core in other places

Reply:
yeah let's hold until it's decided.